packages/jaeger-ui/src/components/DeepDependencies/url.tsx

```diff
@@ -14,8 +14,9 @@
 
 import _isEmpty from 'lodash/isEmpty';
 import memoizeOne from 'memoize-one';
-import queryString from 'query-string';
 import { matchPath } from 'react-router-dom';
+import queryString from 'query-string';
```

Comment:
is it still used?

Reply:
Yes. In `queryString.stringify` function